src/jwtf/src/jwtf.erl

```diff
@@ -70,7 +72,13 @@ encode(Header = {HeaderProps}, Claims, Key) ->
         SignatureOrMac =
             case verification_algorithm(Alg) of
                 {public_key, Algorithm} ->
-                    public_key:sign(Message, Algorithm, Key);
+                    Signature = public_key:sign(Message, Algorithm, Key),
+                    case Alg of
+                        <<"ES", _/binary>> ->
+                            der_to_jose(Alg, Signature);
+                        _ ->
+                            Signature
+                    end;
```

Comment:
Can we factor this into a separate function and use it on lines 279-285 as well?

Reply:
I'm not following. lines 279-285 do the inverse operation. what are you suggesting?

Reply:
My bad, I failed to notice this one uses `der_to_jose` while the other `jose_to_der`. All I can say is "derp derp derp"